types/d3-require/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "unified-signatures": false,
+        "no-unnecessary-generics": false
```

Comment:
Why do you need these?

Reply:
@DanielRosenwasser  this **tslint.json** file is copy from other module, should I fix it?

Reply:
Yes please